app/controllers/api/v1/summaries_controller.rb

```diff
@@ -1,7 +1,22 @@
 class Api::V1::SummariesController < ApplicationController
+
+  rescue_from NoMethodError, with: :not_found_error
+
   def show
-    summary = SummaryService.new.summarize
-    summary_json = JSON.parse(summary.body, symbolize_names: true)
-    render json: SummarySerializer.new(summary_json).serialize_json
+    link = Link.find_by(short: params[:link])
+    if link.summary == nil || link.summary_timestamp < (Time.current - 259200) #3 days in seconds
```

Comment:
I like the way you are storing the shortened links, wondering if we would maybe in the future want to make it longer than three days if this ends up using too many credits

Reply:
yeah I agree 3 days seems a bit too frequent. I have a vague memory of Erin saying something about 3 days in a standup, but I'm not sure